node/network/bridge/src/rx/mod.rs

```diff
@@ -403,6 +403,24 @@ where
 					Ok(v) => v,
 				};
 
+				let maybe_authority =
+					authority_discovery_service.get_authority_ids_by_peer_id(remote).await;
+				if let Some(authority) = maybe_authority {
+					dispatch_validation_events_to_all(
+						vec![NetworkBridgeEvent::UpdatedAuthorityIds(
+							remote.clone(),
+							authority.clone(),
+						)],
+						&mut sender,
+					)
+					.await;
+					dispatch_collation_events_to_all(
+						vec![NetworkBridgeEvent::UpdatedAuthorityIds(remote.clone(), authority)],
```

Comment:
We should notify both the collators and validators right?

Reply:
In principle yes, however, key rotations won't be a problem for collator side since they should be connecting and disconnecting to the assigned validator group which rotates more frequently than a session.

However, I don't think we should be issuing an update on every new message and we should only update if it actually changes.